tools/convert/minimal_config.yaml

```diff
@@ -102,9 +104,14 @@ Collection:
   MaxMemoryPercentage: 75
   MaxAlloc: 0
   DisableRedistribution: false
+  RedistributionDelay: 30s
   ShutdownDelay: 15s
-  EnableTraceLocality: false
+  DisableTraceLocality: false
   HealthCheckTimeout: 3s
+  MaxDropDecisionBatchSize: 1_000
```

Comment:
Should we have these `unpublished` config in the minimal_config.go file?

Reply:
I couldn't decide, so I left them in. If you think they shouldn't be, I can take them out. I don't mind either way.

Reply:
I see. Yes, I would prefer to take them out since we don't really want users to use them yet

Reply:
That was a good catch. Turns out that the minimal config was also including all the obsolete items, so I fixed that too.